deploy/FlapperInit.sol

```diff
@@ -62,67 +63,56 @@ interface FarmLike {
     function setRewardsDuration(uint256) external;
 }
 
-interface FlapperCaller {
-    function file(bytes32, address) external;
-}
-
 struct FlapperUniV2Config {
-    uint256 hop;
     uint256 want;
     address pip;
-    uint256 hump;
-    uint256 bump;
-    address caller;
+    address pair;
+    address splitter;
     bytes32 chainlogKey;
 }
 
 struct SplitterConfig {
+    uint256 hump;
+    uint256 bump;
+    uint256 hop;
     uint256 burn;
     uint256 rewardsDuration;
+    address farm;
+    bytes32 chainlogKey;
```

Comment:
Maybe we should also pass a `previousChainlogKey` here, and if it's not bytes32(0), then we remove it. Thinking that we will need to remove MCD_FLAPPER in the spell.

Reply:
Added to 79a0021c